tests/cli/dorq/workflow/test_logs.py

```diff
@@ -0,0 +1,152 @@
+################################################################################
+# © Copyright 2022-2023 Zapata Computing Inc.
+################################################################################
+"""
+Unit tests for 'orq wf results' glue code.
+"""
+
+from pathlib import Path
+from unittest.mock import Mock, create_autospec
+
+from orquestra.sdk._base.cli._dorq._arg_resolvers import (
+    WFConfigResolver,
+    WFRunIDResolver,
+)
+from orquestra.sdk._base.cli._dorq._dumpers import LogsDumper
+from orquestra.sdk._base.cli._dorq._repos import WorkflowRunRepo
+from orquestra.sdk._base.cli._dorq._ui._presenters import WrappedCorqOutputPresenter
+from orquestra.sdk._base.cli._dorq._workflow import _logs
+
+
+class TestAction:
+    """
+    Test boundaries::
+        [_output.Action]->[arg resolvers]
+                        ->[repos]
+                        ->[dumper]
+                        ->[presenter]
+    """
+
+    class TestDataPassing:
+        """
+        Verifies how we pass variables between subcomponents.
+        """
+
+        @staticmethod
+        def test_no_download_dir():
+            # Given
+            # CLI inputs
+            wf_run_id = "<wf run ID sentinel>"
+            config = "<config sentinel>"
+            download_dir = None
+
+            # Resolved values
+            resolved_id = "<resolved ID>"
+            resolved_config = "<resolved config>"
+
+            # Mocks
+            presenter = create_autospec(WrappedCorqOutputPresenter)
```

Comment:
**question**: would you recommend using `create_autospec` everywhere we would use a `Mock()`?

Reply:
Explained on slack why we found it to be better in those particular cases where we stub entire interface/class